tests/test_signer.py

```diff
@@ -703,6 +703,23 @@ def test_generate(self):
             with self.assertRaises(UnverifiedSignatureError):
                 signer.public_key.verify_signature(sig, b"NOT DATA")
 
+    def test_private_bytes(self):
+        """Test full cycle of generate -> private_bytes -> from_priv_key_uri"""
+        generate_funcs = [
+            CryptoSigner.generate_rsa,
+            CryptoSigner.generate_ecdsa,
+            CryptoSigner.generate_ed25519,
+        ]
```

Comment:
nit: Generate is already tested above. If we want to optimize test speed, we could use the existing test keys in PEMS_DIR / f"{keytype}_private.pem" here.

Reply:
yeah I guess some RSA keygen does take some time...